src/features/unified-share-modal/utils/convertData.js

```diff
@@ -74,20 +55,16 @@ const convertItemResponse = (itemAPIData: ContentSharingItemAPIResponse): Conten
             vanity_name: vanityName,
         } = shared_link;
 
-        const accessLevel = effective_access ? ACCESS_LEVEL_MAP[effective_access] : null;
-        const permissionLevel = effective_permission ? PERMISSION_LEVEL_MAP[effective_permission] : null;
-        const isDownloadAllowed = permissionLevel === PERMISSION_LEVEL_MAP.can_download;
+        const accessLevel = effective_access ? API_TO_USM_ACCESS_LEVEL_MAP[effective_access] : null;
+        const permissionLevel = effective_permission ? API_TO_USM_PERMISSION_LEVEL_MAP[effective_permission] : null;
+        const isDownloadAllowed = permissionLevel === API_TO_USM_PERMISSION_LEVEL_MAP.can_download;
         const canChangeDownload = canChangeAccessLevel && isDownloadAllowed;
         const canChangePassword = canChangeAccessLevel && isPasswordAvailable;
         const canChangeVanityName = canChangeAccessLevel && isVanityNameAvailable;
 
         sharedLink = {
             accessLevel,
-            allowedAccessLevels: {
-                peopleInThisItem: accessLevel === PEOPLE_IN_ITEM,
-                peopleInYourCompany: accessLevel === ANYONE_IN_COMPANY,
-                peopleWithTheLink: accessLevel === ANYONE_WITH_LINK,
-            },
+            allowedAccessLevels: ALLOWED_ACCESS_LEVELS,
```

Comment:
Just confirming based on the comment in constants.js - these default to `true` but may be updated in the future?

Reply:
Yup, exactly. This data is currently not available in the API, so we will be showing all access levels by default.